### PR TITLE
Update SVG package to fix

### PR DESCRIPTION
**Pull request by mikeebowen**
Update SVG package to fix [CVE-2021-26701](https://github.com/advisories/GHSA-ghhp-997w-qr28)

### CODE REVIEW

**Comment by twsouthwick** [comment]
@mikeebowen Update https://github.com/OfficeDev/Open-XML-SDK/blob/main/Directory.Build.props to use net472 as the samples project for framework instead of net46. People are slowly dropping support for older TFMs

File: Directory.Build.props [hunk 1]
```diff
@@ -112,11 +112,11 @@
         .NET Standard target must be first to avoid a ResXFileCodeGenerator issue
         (tracked at https://github.com/dotnet/project-system/issues/1519)
         -->
-        <ProductTargetFrameworks>netstandard1.3;netstandard2.0;net35;net40;net46</ProductTargetFrameworks>
+        <ProductTargetFrameworks>netstandard1.3;netstandard2.0;net35;net40;net472</ProductTargetFrameworks>
```
Comment (twsouthwick) [inline Directory.Build.props:115]: we don't want to change these. Just change the SamplesFramework ones

**Review by twsouthwick — CHANGES_REQUESTED** [review]

**Comment by mikeebowen** [comment]
@twsouthwick , Do you know why the OfficeDev.Open-XML-SDK check hasn't run yet?

**Review by tomjebo — APPROVED** [review]

**Review by twsouthwick — APPROVED** [review]
LGTM! Thanks for just keeping the tfm change to the samples

**Pull request merged**